src/main/res/values/strings.xml

```diff
@@ -120,47 +120,47 @@
   <string name="saving_details">Saving details</string>
   <string name="screenshots">Screenshots</string>
   <string name="search">Search</string>
-  <string name="select_the_mirror">Select the mirror</string>
-  <string name="show_more">Show more</string>
-  <string name="show_older_releases">Show older releases</string>
+  <string name="select_the_mirror">Select a mirror</string>
+  <string name="show_more">More</string>
+  <string name="show_older_releases">Show older versions</string>
   <string name="skip">Skip</string>
   <string name="socks_proxy">SOCKS proxy</string>
-  <string name="sort_order">Sort order</string>
+  <string name="sort_order">Sorting order</string>
   <string name="source_code">Source code</string>
   <string name="suggested">Suggested</string>
   <string name="sync_repositories">Sync repositories</string>
-  <string name="sync_repositories_automatically">Sync repositories automatically</string>
-  <string name="syncing">Syncing</string>
-  <string name="syncing_format">Syncing %s</string>
+  <string name="sync_repositories_automatically">Automatically</string>
+  <string name="syncing">Syncing…</string>
+  <string name="syncing_format">Syncing %s…</string>
   <string name="tap_to_install_description">Tap to install.</string>
   <string name="theme">Theme</string>
   <string name="uninstall">Uninstall</string>
   <string name="unknown">Unknown</string>
   <string name="unknown_error_description">Unknown error.</string>
   <string name="unknown_format">Unknown: %s</string>
   <string name="unsigned">Unsigned</string>
-  <string name="unsigned_description">Unsigned. The list of applications could not be verified. Be careful with
-    applications downloaded from unsigned repositories.</string>
+  <string name="unsigned_description">Unsigned. Could not verify the application list. Be careful downloading
+    application from unsigned repositories.</string>
   <string name="unstable_updates">Unstable updates</string>
-  <string name="unstable_updates_summary">Suggest updates to unstable versions</string>
+  <string name="unstable_updates_summary">Suggest upgrades to unstable versions</string>
```

Comment:
I'm using "update" word instead of "upgrade". I don't want to use different names for the same processes.

Reply:
Can it be "Suggest unstable versions"?

Reply:
Well, the full preference text was something like:

Title: **Unstable updates**
Subtitle: **Suggest updates to unstable versions**

(Notice how you changed "updates" to "upgrades" only in one place)

It's possible to make it so, but then it will be

Title: **Unstable updates**
Subtitle: **Suggest unstable versions**

Doesn't seem like necessary change.

Reply:
In my head "updates" are only syncing repos, and upgrades is installing new versions.
Avoided it in the second sentence to make sure it actually explains something for those that don't know what "updates" entail.

Reply:
> In my head "updates" are only syncing repos, and upgrades is installing new versions.

That's not necessarily true. For example, in Gentoo "sync" is syncing (this process called "update" in Debian or "refresh" in Arch) and "update" is.. updaing? (this called "upgrade" in Debian/Arch).